paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerModule.kt

```diff
@@ -28,7 +28,13 @@ internal object FlowControllerModule {
     @Provides
     @Singleton
     @Named(PRODUCT_USAGE)
-    fun provideProductUsageTokens() = setOf("PaymentSheet.FlowController")
+    fun provideProductUsageTokens(viewModel: FlowControllerViewModel): Set<String> {
+        val deferred = viewModel.state?.stripeIntent?.clientSecret == null
```

Comment:
`viewModel.state` will always be `null` when this is called during `DefaultFlowController` construction.